content/docs/latest/howtorelease_27362106.md

```diff
@@ -254,6 +256,7 @@ hive-standalone-metastore-X.Y.Z-src.tar.gz: OK
 4. Check that release file looks ok -- e.g., install it and run examples from tutorial.
 5. Setup your PGP keys for signing the release, if you don't have them already.
 	1. See <https://www.apache.org/dev/release-signing.html>, <https://www.apache.org/dev/openpgp.html>.
+    2. You may need PMC privileges to do this step – if you do not have such privileges, please ping a [PMC member](http://hive.apache.org/people.html) to do this for you. 
```

Comment:
From the moment that we allow & encourage committers to serve as release managers I think it makes sense to lift these limitations and give them permissions to the dist area.

I started a discussion about that on the dev list. If the rest of the PMC are onboard with the idea we can remove entirely this sentence from the release guide.

Reply:
The PMC approved the changes and https://issues.apache.org/jira/browse/INFRA-27113 is mostly done so we can now remove the sentence from everywhere ;)